sdk/python/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb

```diff
@@ -639,6 +654,87 @@
    ]
   },
   {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4.2.1 Individual runs with models from Hugging Face (Preview)\n",
+    "\n",
+    "In addition to the model algorithms supported natively by AutoML, you can launch individual runs to explore any model algorithm from HuggingFace that supports image classification. Please refer to this [documentation](https://huggingface.co/models?pipeline_tag=image-classification) for the list of models.\n",
```

Comment:
we only support transformers library

Reply:
use this link instead: https://huggingface.co/models?pipeline_tag=image-classification&library=transformers

Reply:
applies to both classification notebooks

Reply:
Good catch. Will update it.